docs/conf.py

```diff
@@ -38,10 +38,7 @@
 templates_path = ["_templates"]
 
 # The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-#
-# source_suffix = ['.rst', '.md']
-source_suffix = ".md"
```

Comment:
The curent docs source files are RST, but this line was militing to only looks for markdown files. Additionaly, some of these files includes the markdown from the repo root:

https://github.com/namespace-ee/django-datadog-logger/blob/3870d2c1d71240cd6c795e45b8153644c63a3f75/docs/readme.rst#L1

Resulting in invalid rendering:

<img width="1095" alt="image" src="https://github.com/namespace-ee/django-datadog-logger/assets/861044/36287b55-e6f7-4031-bd94-04365dbb6f78">

https://browniebroke-django-datadog-logger--1.org.readthedocs.build/en/1/readme.html

Reply:
They indeed were rst, but I've started to convert them to markdown in main already.